### PR TITLE
loccheckin 20200924

### PR DESCRIPTION
**Pull request by cristianosuzuki77**
4 main updates in this PR

1 Pending fixed for the following issues: 

- https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1163530
- https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1180932
- https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1178477
- https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1180932
- https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1174024

2 Update to the extra dots flagged in PR https://github.com/dotnet/roslyn/pull/47532.
Note: Per translators, some of the dots are expected (depending on the language and the construct of the sentence, the full stop is added).

3 Additional fixes for placeholders missing quotation or with the incorrect language quotes.

4. Additional new translations.

### CODE REVIEW

**Comment by wli3** [comment]
@JoeRobich could you review this change?

File: src/Analyzers/CSharp/Analyzers/xlf/CSharpAnalyzersResources.pl.xlf [hunk 1]
```diff
@@ -274,7 +274,7 @@
       </trans-unit>
       <trans-unit id="typeof_can_be_converted_ to_nameof">
         <source>'typeof' can be converted  to 'nameof'</source>
-        <target state="new">'typeof' can be converted  to 'nameof'</target>
+        <target state="translated">Element „typeof” można przekonwertować na element „nameof”</target>
```
Comment (Youssef1313) [inline src/Analyzers/CSharp/Analyzers/xlf/CSharpAnalyzersResources.pl.xlf:277]: Is the use of `„ ”` intended?

File: src/Analyzers/CSharp/Analyzers/xlf/CSharpAnalyzersResources.ko.xlf [hunk 2]
```diff
@@ -274,7 +274,7 @@
       </trans-unit>
       <trans-unit id="typeof_can_be_converted_ to_nameof">
         <source>'typeof' can be converted  to 'nameof'</source>
-        <target state="new">'typeof' can be converted  to 'nameof'</target>
+        <target state="translated">'typeof'를 'nameof'로 변환할 수 있습니다.</target>
```
Comment (Youssef1313) [inline src/Analyzers/CSharp/Analyzers/xlf/CSharpAnalyzersResources.ko.xlf:277]: Whatever the language is, it's weird not to have a space after `'typeof'` and `'nameof'`.

**Comment by JoeRobich** [comment]
@wli3 Do we want these in 16.8? master is currently 16.9.

**Comment by wli3** [comment]
ah, ya. it should go 16.8

**Comment by JoeRobich** [comment]
@cristianosuzuki77 Can you rebase your changes on `release/dev16.8-vs-deps`?

File: src/Analyzers/CSharp/Analyzers/xlf/CSharpAnalyzersResources.pl.xlf (hunk 1 above)
Comment (cristianosuzuki77) [reply src/Analyzers/CSharp/Analyzers/xlf/CSharpAnalyzersResources.pl.xlf:277]: Yes

File: src/Analyzers/CSharp/Analyzers/xlf/CSharpAnalyzersResources.ko.xlf (hunk 2 above)
Comment (cristianosuzuki77) [reply src/Analyzers/CSharp/Analyzers/xlf/CSharpAnalyzersResources.ko.xlf:277]: Space is not common for   Korean and Japanese.

**Comment by cristianosuzuki77** [comment]
@JoeRobich the loc tool is scanning master. If you need this to go to release/dev16.8-vs-deps could you accept this in master and I will re-configure the loc tool to start scanning release/dev16.8-vs-deps and I'll perform another new PR after the scanning is complete and any new string is translated?

**Comment by wli3** [comment]
@cristianosuzuki77 will this change be part of the next loc pass? If so, we can just wait for the next loc pass.

**Comment by cristianosuzuki77** [comment]
@wli3 I saw your request late Friday, before commenting. So yes, it will. And I just scanned release/dev16.8 and have not seeing any new file that wasn't already scanned in master. I will PR to release/dev16.8 once any pending string from it is returned.

**Comment by JoeRobich** [comment]
@cristianosuzuki77 Sorry I didn't get to this before other .xlf changes sneaked in. There are now conflicts in this PR but no way to resolve them.

**Comment by cristianosuzuki77** [comment]
These strings will be re-submitted once the loc tool starts scanning the master branch.
Closing this PR instead.

**Pull request closed without merging**